rust/connlib/tunnel/src/lib.rs

```diff
@@ -352,10 +352,10 @@ where
         if self.stats_timer.poll_tick(cx).is_ready() {
             let (node_stats, conn_stats) = self.node.stats();
 
-            tracing::info!(target: "connlib::stats", "{node_stats:?}");
+            tracing::debug!(target: "connlib::stats", "{node_stats:?}");
```

Comment:
@thomaseizinger I think this prints every 10 seconds? That's probably a bit too much for production gateways / clients. Could this be debug! instead? We can always override the log filter in clients and gateways to set our own.

Reply:
I just picked 10s arbitrarily, happy to change it. Maybe 1 minute would be better? "debug" is probably a fair call yeah.

Reply:
One minute sounds good, I'll update it. Yeah I'm about to cut a new release in a minute so was hoping to resolve this before it goes out to users.